stable/prometheus-blackbox-exporter/templates/role.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ include "prometheus-blackbox-exporter.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "prometheus-blackbox-exporter.chart" . }}
+  name: {{ template "prometheus-blackbox-exporter.fullname" . }}
```

Comment:
maybe add -psp to the name so that we know at first glance what it is? Same for rolebinding.